Storage/SQL/CommandSyncTable.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CommandSyncTable.swift
+//  Client
+//
+//  Created by Emily Toop on 6/29/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import Shared
+
+class CommandSyncTable<T>: GenericTable<SyncCommand> {
+    override var name: String { return "commands" }
+    override var version: Int { return 1 }
+
+    override var rows: String { return join(",", [
+        "guid TEXT PRIMARY KEY",
+        "client_guid TEXT REFERENCES clients(guid)",
+        "url TEXT",
+        "title TEXT",
+        "action TEXT NOT NULL",
+        "last_used INTEGER",
+        ])
+    }
+
+
+    override func getInsertAndArgs(inout item: SyncCommand) -> (String, [AnyObject?])? {
+        var args = [AnyObject?]()
+        args.append(item.guid)
+        args.append(item.client)
+        args.append(item.url)
+        args.append(item.title)
+        args.append(item.action)
+        args.append(NSNumber(unsignedLongLong: item.modified))
+        return ("INSERT INTO \(name) (guid, client_guid, url, title, action, last_used) VALUES (?, ?, ?, ?, ?, ?)", args)
+    }
+
+    override func getDeleteAndArgs(inout item: SyncCommand?) -> (String, [AnyObject?])? {
+        if let item = item {
+            return ("DELETE FROM \(name) WHERE guid = ?", [item.guid])
```

Comment:
I believe this will crash with an unsafe bit cast error. You need to break it out somehow: either

```
[item.guid as AnyObject]
```

or

```
let args: Args = [item.guid]
…
```